packages/js/src/plugins/auctionHouseModule/createAuctionHouse.ts

```diff
@@ -149,5 +156,41 @@ export const createAuctionHouseBuilder = (
       ),
       signers: [payer],
       key: params.instructionKey ?? 'createAuctionHouse',
+    })
+    .when(Boolean(params.auctioneerAuthority?.publicKey), (builder) => {
+      const auctioneerAuthority = params.auctioneerAuthority
+        ?.publicKey as PublicKey;
+      const ahAuctioneerPda = findAuctioneerPda(
+        auctionHouse,
+        auctioneerAuthority
+      );
+
+      const scopes = params.auctioneerScopes ?? [
+        AuthorityScope.Deposit,
+        AuthorityScope.Buy,
+        AuthorityScope.PublicBuy,
+        AuthorityScope.ExecuteSale,
+        AuthorityScope.Sell,
+        AuthorityScope.Cancel,
+        AuthorityScope.Withdraw,
+      ];
+
+      builder.add({
+        instruction: createDelegateAuctioneerInstruction(
+          {
+            auctionHouse,
+            authority,
+            auctioneerAuthority,
+            ahAuctioneerPda,
+          },
+          {
+            scopes,
+          }
+        ),
+        signers: [payer],
```

Comment:
This should not be `payer` but `authority`.

See https://github.com/metaplex-foundation/metaplex-program-library/blob/cac9cfd042cc5a16f50d635074555392d3b9044e/auction-house/js/src/generated/instructions/delegateAuctioneer.ts#L85